style.css

```diff
@@ -44,7 +44,8 @@ body {
 .no-signal {
   width: 100%; 
   height: 100%; 
-  position: relative;
+  position: fixed; /* changed to fixed */
```

Comment:
Great method to fix the background issue 💯 🦜 🥳

Reply:
Amazing, thank you for fixing this